syft/file/classfication_cataloger_test.go

```diff
@@ -0,0 +1,164 @@
+package file
+
+import (
+	"testing"
+
+	"github.com/anchore/syft/syft/source"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClassifierCataloger_DefaultClassifiers_PositiveCases(t *testing.T) {
+	tests := []struct {
+		name           string
+		fixtureDir     string
+		location       string
+		expected       []Classification
+		constructorErr bool
+		catalogErr     bool
```

Comment:
What is the function of `catalogErr`?

Reply:
copy-pasta from another test, I'll remove